tests/storage/clone.go

```diff
@@ -120,6 +175,35 @@ var _ = SIGDescribe("[Serial]VirtualMachineClone Tests", func() {
 		return out
 	}
 
+	createSnapshotAndWaitForReady := func(vmName, vmNamespace string) *v1alpha1.VirtualMachineSnapshot {
+		var err error
+
+		snapshot := &v1alpha1.VirtualMachineSnapshot{
```

Comment:
dont you want to use here the func you create `createSnapshot` since its the same code?

Reply:
Of course :O!
thanks for catching that